java/kafka/consumer/src/main/java/KafkaConsumerExample.java

```diff
@@ -22,44 +21,68 @@
+        TracingSystem tracingSystem;
+        tracingSystem = TracingSystem.forValue(config.getTracingSystem());
 
-            props.put(ConsumerConfig.INTERCEPTOR_CLASSES_CONFIG, TracingConsumerInterceptor.class.getName());
+        if (tracingSystem != null) {
+            if (tracingSystem == KafkaConsumerExample.TracingSystem.JAEGER) {
+                Tracer tracer = Configuration.fromEnv().getTracer();
+                GlobalTracer.registerIfAbsent(tracer);
+                props.put(ConsumerConfig.INTERCEPTOR_CLASSES_CONFIG, io.opentracing.contrib.kafka.TracingConsumerInterceptor.class.getName());
+            } else if (tracingSystem == KafkaConsumerExample.TracingSystem.OPENTELEMETRY) {
+                props.put(ConsumerConfig.INTERCEPTOR_CLASSES_CONFIG, io.opentelemetry.instrumentation.kafkaclients.TracingConsumerInterceptor.class.getName());
+            } else {
+                log.error("Error: TRACING_SYSTEM {} is not recognized or supported!", config.getTracingSystem());
+            }
         }
 
-        boolean commit = !Boolean.parseBoolean(config.getEnableAutoCommit());
-        KafkaConsumer consumer = new KafkaConsumer(props);
-        consumer.subscribe(Collections.singletonList(config.getTopic()));
 
-        while (receivedMsgs < config.getMessageCount()) {
-            ConsumerRecords<String, String> records = consumer.poll(Duration.ofMillis(Long.MAX_VALUE));
-            for (ConsumerRecord<String, String> record : records) {
-                log.info("Received message:");
-                log.info("\tpartition: {}", record.partition());
-                log.info("\toffset: {}", record.offset());
-                log.info("\tvalue: {}", record.value());
-                if (record.headers() != null) {
-                    log.info("\theaders: ");
-                    for (Header header : record.headers()) {
-                        log.info("\t\tkey: {}, value: {}", header.key(), new String(header.value()));
+            boolean commit = !Boolean.parseBoolean(config.getEnableAutoCommit());
+            KafkaConsumer consumer = new KafkaConsumer(props);
+            consumer.subscribe(Collections.singletonList(config.getTopic()));
+
+            while (receivedMsgs < config.getMessageCount()) {
+                ConsumerRecords<String, String> records = consumer.poll(Duration.ofMillis(Long.MAX_VALUE));
+                for (ConsumerRecord<String, String> record : records) {
+                    log.info("Received message:");
+                    log.info("\tpartition: {}", record.partition());
+                    log.info("\toffset: {}", record.offset());
+                    log.info("\tvalue: {}", record.value());
+                    if (record.headers() != null) {
+                        log.info("\theaders: ");
+                        for (Header header : record.headers()) {
+                            log.info("\t\tkey: {}, value: {}", header.key(), new String(header.value()));
+                        }
```

Comment:
This code block should be aligned/formatted to the code above, I think these formatting changes were added by mistake